std/pdb.zig

```diff
@@ -660,23 +660,24 @@ const MsfStream = struct {
         return size;
     }
 
-    fn seekForward(self: *MsfStream, len: usize) !void {
+    // XXX: The `len` parameter should be signed
```

Comment:
Why?

Reply:
Because all the other `seekForward` do so. Even though negative offsets are not used in the pdb parsers I found it a bit out of place.

Reply:
ah okay.
We should probably rename it to `seekBy` or similar if it takes a signed number.